### DISCUSSION TITLE
Floating Action Buttons - Multiple Chat Buttons

### DISCUSSION THREAD

**Discussion by raedweb** [TDRA-ae/aegov-dls#54] [General]
Dear Folks and Support Team,

As per the guideline for Floating action button, Limit the number of floating elements. The design system restricts the use of floating action buttons to a maximum of two (2) on a website.
[Chat with us](https://designsystem.gov.ae/#): For initiating a conversation or getting support.
[Customer Pulse](https://designsystem.gov.ae/#): For giving feedback or taking a survey.

**Need your Advice:**
We have two chat channel one for Livechat an dother Chatbot, so do you think is okay if we make the Chat action button on click to show a modal/popup that has two buttons 
![chat-website](https://github.com/user-attachments/assets/165dab3c-8f81-42ba-8701-f6cd53ecc0f1)

**Comment by moiz-conceptlz** [comment]
Hello @raedweb 

A simple, effective and well-thought-through user experience is a critical aspect of the UAE Design System. It's not just about using components but also about planning a user journey and experience that is frictionless, easy, and useful.

Therefore, I personally find it to be flawed to have the user click on a chat icon and then further be presented with the 2 options - which is to chat with a human or to chat with a bot. When a user clicks on a chat icon - they expect to chat! No matter if it is a Human or an AI-driven bot, the intention of the user is to get answers to their questions. Why would someone knowingly want to chat with a Bot!? what happens if a bot is not able to help them - will they start the user journey again? 

If you pick any experience of a chat window - which today, mostly starts out with a bot providing answers - usually is a single click and the ability for a user to start a conversation. The intelligence of the bot determines whether or not they would like to talk to a human. Should a user get their answer via a bot, it ends the journey.

The purpose of restricting 2 buttons for a Floating action button is not to provide the user with 2 more options. What you are referring to is a workaround to a requirement. However, the design system is trying to focus on improving the user journey and reducing the need for floating actions altogether. 

I recommend finding a solution that can create a better user experience. Should you not have the option to do so, a strong justification would be required as to why your approach is the way forward. Technology-wise, there is certainly no challenge as it is a very common practice to use systems that provide such functionality.

**Comment by raedweb** [comment]
Thank you for clarifications
